CMSgov/mint-app
# [EASI-2216] FE work for Read Only Documents

# EASI-2216

## Changes and Description

- Creates new Participants and Providers Read Only component
- Adds test and snaps

## Acceptance Criteria

- Remove document should only be allowed with team members with edit access 
- Side contact bar should be hidden
- Testing

## How to test this change

1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Click `Documents` on sidebar
3. Ensure component is displayed correctly and matches figma




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
